src/zenhub.py

```diff
@@ -164,5 +164,38 @@ def update_issue(self, points=None, pipeline=None, pipeline_pos=None, to_epic=Fa
         logger.info(f'Finished updating {self.issue} in repo {self.repo_name}')
 
 
+class ZenHubIssue(Issue):
+
+    def __init__(self, key: str = None, repo_name: str = None, response: dict = None):
+        """
+        Create an Issue object from an issue key and repo name or from a portion of an API response
```

Comment:
I would specify what API (we're dealing with three APIs in this repo). I suppose you mean the GitHub, but it's easier to understand if it isname here.

Reply:
specified name